cpp/include/cuspatial/experimental/detail/trajectory_distances_and_speeds.cuh

```diff
@@ -122,8 +122,8 @@ OutputIt trajectory_distances_and_speeds(IndexT num_trajectories,
                           auto time_s = static_cast<T>(time_d.count()) *
                                         static_cast<T>(Period::num) / static_cast<T>(Period::den);
                           T dist_km   = thrust::get<1>(a) + thrust::get<1>(b);
-                          T dist_m    = dist_km * T{1000.0};  // km to m
-                          T speed_m_s = dist_m / time_s;      // m/ms to m/s
+                          T dist_m    = dist_km* T{1000.0};  // km to m
```

Comment:
This seems like a wierd binary op formatting?

Reply:
I agree this is weird. Perhaps we need a different clang-format rule. Can you look into this?

Reply:
I briefly searched through https://clang.llvm.org/docs/ClangFormatStyleOptions.html but didn't immediately see the option that would control this. I think this would be better spelled as `static_cast<T>(1000.0)`. What do you think of that option?

Reply:
Cross-referencing this thread: https://github.com/rapidsai/cugraph-ops/pull/466#discussion_r1168977855

Reply:
I've triaged this and found that the bug does not exist in 15.0.7 or 17.0.0 (dev branch), but does exist in 16.0.0, 16.0.1, and 16.0.2 (the latter could change since it is not yet released). For now, I'm going to update this PR to 15.0.7.